README.md

```diff
@@ -90,6 +94,11 @@ For development you can also watch source files for changes and automatically li
 
     make watch
 
+## Web server
+
+Web server can run through the command line:
```

Comment:
Does this need to be run with a web server for development? I'm not sure it does, or this documentation would already be included…